docs: rename data to JSON and dataframe to table (#12352)
langflow-ai/langflow: docs/docs/Develop/data-types.mdx

Reviewer: JSON or Json?

```suggestion
data_obj = JSON(
```

* If two components have incompatible data types, you can use a processing component like the [**Type Convert** component](/type-convert) to convert the data between components.
:::

## JSON

:::tip

> Prior to version 1.9.0, the JSON port was called `Data`.

Reviewer: Optional

```suggestion
In Langflow version 1.9.0, the `Data` type and port were renamed to `JSON`.
```

}
```

## Table

:::tip

> Prior to version 1.9.0, the Table port was called `DataFrame`.

Reviewer: ```suggestion
In Langflow version 1.9.0, the `DataFrame` type and port were renamed to `Table`.
```
